src/v2/Apps/Auction/Routes/Bid/__tests__/AuctionBidRoute.jest.tsx

```diff
@@ -241,7 +241,7 @@ describe("AuctionBidRoute", () => {
     ;(wrapper.find("Select").props() as any).onSelect("1000")
 
     expect(spy).toHaveBeenCalledWith({
-      bidderID: "<Bidder-mock-id-7>",
+      bidderID: "<Bidder-mock-id-6>",
```

Comment:
This felt like a curious spec update needed.

Reply:
This is fine, but might be worth while mocking `bidderID` from `getWrapper`